plone/outputfilters/tests/test_picture_variants.py

```diff
@@ -246,10 +245,5 @@ def test_parsing_with_nonexisting_srcset(self):
         )
         res = self.parser(text)
         self.assertTrue(res)
-        text_out = """
```

Comment:
@MrTango either you were thinking on testing something here, is a copy&paste leftover, or something was lost, as this line is not being used (`flake8` made me notice it).